resources/analysisTools/copyNumberEstimationWorkflow/phasing.sh

```diff
@@ -20,12 +20,12 @@ then
 
     CHR_NR="$CHR_PREFIX${CHR_NAME:?CHR_NAME is not set}"
 
-    $SAMTOOLS_BINARY mpileup "$CNV_MPILEUP_OPTS" -u \
+    $SAMTOOLS_BINARY mpileup $CNV_MPILEUP_OPTS -u \
```

Comment:
Did you try `${CNV_MPILEUP_OPTS[@]}"? It expands to the elements of a bash array and quotes each. This would also allow for unquoted elements that include whitespaces.

Reply:
`$SAMTOOLS_BINARY mpileup "${CNV_MPILEUP_OPTS[@]}" -u -f` translates into `samtools mpileup '-A -R -B -Q 0 -q 1 -I ' -u -f /icgc/... ` The `'` is recognized as a value and generates wrong parameters as input.

Reply:
@GWarsow Here is the previous comment about `CNV_MPILEUP_OPTS` issue.